tools/convert/rs-convert.cpp

```diff
@@ -178,19 +245,22 @@ int main(int argc, char** argv) try
         {
             int posP = static_cast<int>(posLast * 100. / duration.count());
 
-            if (posP > progress) {
+            if (posP > progress)
+            {
                 progress = posP;
                 cout << posP << "%" << "\r" << flush;
             }
 
             const uint64_t posCurr = playback.get_position();
-            if (static_cast<int64_t>(posCurr - posLast) < 0) {
+            if (static_cast<int64_t>(posCurr - posLast) < 0)
```

Comment:
Please change this loop, too -- to agree with the names and logic of the first loop